From 46382e01243e30156660f6efe84a9afee0c5c1aa Mon Sep 17 00:00:00 2001
From: Antoine Martin <dev@ayakael.net>
Date: Sun, 28 Aug 2022 05:19:20 +0000
Subject: [PATCH 2/2] aspnetcore_set-local-repo

Patch necessary for stage0, as each component requires the artifacts
from previously built component.  This patch (along with a prepare step
in aport) adds a "local" nupkg repository that artifacts are pushed to
at end of component build

---
 src/aspnetcore/NuGet.config | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/runtime/NuGet.config b/src/runtime/NuGet.config
index 71e79dd377..c14909e175 100644
--- a/src/runtime/NuGet.config
+++ b/src/runtime/NuGet.config
@@ -7,6 +7,7 @@
   </fallbackPackageFolders>
   <packageSources>
     <clear />
+    <add key="local" value="@@PACKAGESDIR@@" />
     <!--Begin: Package sources managed by Dependency Flow automation. Do not edit the sources below.-->
     <!--  Begin: Package sources from dotnet-emsdk -->
     <add key="darc-pub-dotnet-emsdk-6b7d1f4" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-emsdk-6b7d1f47/nuget/v3/index.json" />
diff --git a/src/aspnetcore/NuGet.config b/src/aspnetcore/NuGet.config
index a26b4ca86..1f40b8f06 100644
--- a/src/aspnetcore/NuGet.config
+++ b/src/aspnetcore/NuGet.config
@@ -2,6 +2,7 @@
 <configuration>
   <packageSources>
     <clear />
+    <add key="local" value="@@PACKAGESDIR@@" />
     <!--Begin: Package sources managed by Dependency Flow automation. Do not edit the sources below.-->
     <!--  Begin: Package sources from dotnet-efcore -->
     <add key="darc-int-dotnet-efcore-865c6a8" value="https://pkgs.dev.azure.com/dnceng/internal/_packaging/darc-int-dotnet-efcore-865c6a89/nuget/v3/index.json" />
diff --git a/src/installer/NuGet.config b/src/installer/NuGet.config
index 8cac51c22..c07e44968 100644
--- a/src/installer/NuGet.config
+++ b/src/installer/NuGet.config
@@ -5,6 +5,7 @@
   </solution>
   <packageSources>
     <clear />
+    <add key="local" value="@@PACKAGESDIR@@" />
     <!--Begin: Package sources managed by Dependency Flow automation. Do not edit the sources below.-->
     <!--  Begin: Package sources from dotnet-windowsdesktop -->
     <add key="darc-pub-dotnet-windowsdesktop-8a65067" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-windowsdesktop-8a650675/nuget/v3/index.json" />
diff --git a/src/roslyn/NuGet.config b/src/roslyn/NuGet.config
index a26b4ca86..1f40b8f06 100644
--- a/src/roslyn/NuGet.config
+++ b/src/roslyn/NuGet.config
@@ -3,6 +3,7 @@
 <configuration>
   <packageSources>
     <clear />
+    <add key="local" value="@@PACKAGESDIR@@" />
     <add key="dotnet-eng" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-eng/nuget/v3/index.json" />
     <add key="dotnet-tools" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-tools/nuget/v3/index.json" />
     <add key="dotnet5" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet5/nuget/v3/index.json" />
diff --git a/src/sdk/NuGet.config b/src/sdk/NuGet.config
index 2773a4d0b..9aab732f7 100644
--- a/src/sdk/NuGet.config
+++ b/src/sdk/NuGet.config
@@ -2,6 +2,7 @@
 <configuration>
   <packageSources>
     <clear />
+    <add key="local" value="@@PACKAGESDIR@@" />
     <!--Begin: Package sources managed by Dependency Flow automation. Do not edit the sources below.-->
     <!--  Begin: Package sources from dotnet-aspnetcore -->
     <add key="darc-int-dotnet-aspnetcore-6a01dd1" value="https://pkgs.dev.azure.com/dnceng/internal/_packaging/darc-int-dotnet-aspnetcore-6a01dd1e/nuget/v3/index.json" />
-- 
2.37.1
